pom.xml

```diff
@@ -405,6 +405,9 @@
         <artifactId>maven-compiler-plugin</artifactId>
         <version>3.10.1</version>
         <configuration>
+          <compilerArgs>
+            <arg>-parameters</arg>
```

Comment:
`-parameters` option will include the parameter names when calling `getParameters()`

Reply:
Nice! I was surprised that this information was available at all, and also didn't realize that a special compiler flag was necessary to gather it.